.buildkite/ftr_configs.yml

```diff
@@ -115,9 +115,15 @@ enabled:
   - test/functional/apps/discover/classic/config.ts
   - test/functional/apps/discover/embeddable/config.ts
   - test/functional/apps/discover/group1/config.ts
-  - test/functional/apps/discover/group2/config.ts
+  - test/functional/apps/discover/group2_data_grid1/config.ts
+  - test/functional/apps/discover/group2_data_grid2/config.ts
+  - test/functional/apps/discover/group2_data_grid3/config.ts
   - test/functional/apps/discover/group3/config.ts
   - test/functional/apps/discover/group4/config.ts
+  - test/functional/apps/discover/group5/config.ts
+  - test/functional/apps/discover/group6/config.ts
+  - test/functional/apps/discover/group7/config.ts
+  - test/functional/apps/discover/group8/config.ts
```

Comment:
Just curious, how much quicker is it, generally speaking?

Reply:
@wayneseymour These are the times for the new groups:

<img width="500" alt="Screenshot 2024-04-16 at 19 52 24" src="https://github.com/elastic/kibana/assets/1415710/3dda61cc-1ddd-4e0a-9d45-d3a896906784">

https://buildkite.com/elastic/kibana-pull-request/builds/201107#018ea964-5b6b-436c-9b49-b2daf1e30abd

And these tests were inside other groups previously. So it's somewhat around 40-50%.

Reply:
awesome